migrate/migrations/20200922140909-session-last-access-time.sql

```diff
@@ -0,0 +1,9 @@
+-- +migrate Up
+ALTER TABLE auth_user_sessions
+    ALTER COLUMN last_access_at SET NOT NULL,
+    ALTER COLUMN last_access_at SET DEFAULT now();
```

Comment:
`DEFAULT` needs to be set before `NOT NULL`. Production tables will have null values causing this migration to fail otherwise.

For migrate down, keep the order you have (drop not null, then drop default).

Reply:
addressed, changes made as requested.